aptos-move/aptos-release-builder/data/release-v1.4.yaml

```diff
@@ -6,19 +6,8 @@ update_sequence:
   - Framework:
       bytecode_version: 6
       git_hash: ~
-  - DefaultGas
   - FeatureFlag:
       enabled:
-        - code_dependency_check
-        - treat_friend_as_private
-        - sha512_and_ripe_md160_natives
-        - aptos_std_chain_id_natives
-        - v_m_binary_format_v6
-        - multi_ed25519_pk_validate_v2_natives
-        - blake2b256_native
-        - resource_groups
-        - multisig_accounts
-        - delegation_pools
```

Comment:
hmm, this looks suspicious, is it because those feature flags being removed i.e. default enabled?